src/xdocs/config_annotation.xml

```diff
@@ -142,13 +142,20 @@ public String getNameIfPresent() { ... }
 
       <subsection name="Examples" id="AnnotationLocation_Examples">
         <p>
-          Example to allow multiple annotations on the same line
+          Default configuration:
         </p>
         <source>
-@SuppressWarnings("deprecation") @Mock DataLoader loader; // no violations
+&lt;module name=&quot;AnnotationLocation&quot;/&gt;
+        </source>
+        <p>
+          Example for default configuration
+        </p>
+        <source>
+@Override public int hashCode() { ... } // no violations
```

Comment:
use `ok` instead of using the word violation to prevent any confusion.

Reply:
Done